docs/js/python.md

```diff
@@ -1,14 +1,56 @@
-# Python Converter
+# Python Support
 
-**This is work in progress**
+**Python support in MakeCode is experimental**
+
+We currently refer to TypeScript version supported by MakeCode as MakeCode TypeScript (MTS)
```

Comment:
MakeCode Typescript(MTS)? It is referred as Static Typescript all over the place

Reply:
well, if it's Static TypeScript, it's also Static Python. Changed.